lua/neotest-haskell/init.lua

```diff
@@ -17,6 +17,9 @@ local is_test_file = base.is_test_file
 ---@type fun(name:string):boolean
 local filter_dir = base.filter_dir
 
+---@type table
+local build_tools = { 'stack', 'cabal' }
```

Comment:
```suggestion
local supported_build_tools = { 'stack', 'cabal' }
local build_tools = supported_build_tools
```
See my next suggestion for context.